scripts/qa_crowbarsetup.sh

```diff
@@ -2284,6 +2336,15 @@ function custom_configuration
             if iscloudver 7plus ; then
                 proposal_set_value nova default "['attributes']['nova']['metadata']['vendordata']['json']" "'{\"custom-key\": \"custom-value\"}'"
             fi
+
+            if iscloudver 7plus && [[ $want_ironic = 1 ]] ; then
+                if [[ $hacloud = 1 ]] ; then
+                    proposal_set_value nova default "['deployment']['nova']['elements']['nova-compute-ironic']" "['cluster:$clusternameservices']"
```

Comment:
Failed to validate proposal: Role nova-compute-ironic does not accept clusters.

Reply:
Fixed